docs-src/cloud/cloud-ops-api.md

```diff
@@ -0,0 +1,110 @@
+
+- [Temporal Cloud user account](/cloud/get-started)
+- [API Key](/cloud/tcld/apikey/create) for authentication
+- [Protocol Buffers](https://github.com/protocolbuffers/protobuf/releases)
+
+### Compile the API
+
+Download the gRPC protobufs from the [Cloud Ops API repository](https://github.com/temporalio/api-cloud/tree/main/temporal/api/cloud).
+
+Use [grpc](https://grpc.io/docs/) to compile and generate code in your preferred [programming language](https://grpc.io/docs/#official-support).
+
+1. **Clone the Temporal Cloud API repository:**
+   ```command
+   git clone https://github.com/temporalio/api-cloud.git
+   cd api-cloud
+   ```
+
+2. **Copy Protobuf files:**
+   - Navigate to the `temporal` directory.
+   - Copy the protobuf files to your project directory.
+
+3. **Compile the Protobuf files:**
+   1. The following is an example of how to use the generated code for Python; however, this approach can be adapted for other supported programming languages:
+   ```python
+   python -m grpc_tools.protoc -I./ --python_out=./ --grpc_python_out=./ *.proto
+   ```
+   - `-I./` specifies the directory of the `.proto` files.
+   - `--python_out=./` sets the output directory for generated Python classes.
+   - `--grpc_python_out=./` sets the output directory for generated gRPC service classes.
+   - `*.proto` processes all `.proto` files.
+
+## Use the generated code
+
+After compiling the Protobuf files, you will have generated code files in your project directory.
+These files enable interaction with the Temporal Cloud API in your chosen programming language.
+
+The following is an example of how to use the generated code for Python; however, this approach can be adapted for other programming languages:
+
+1. **Import the Generated Files:**
+   - Locate the Python files (.py) generated in your project directory.
+   - Import these files into your Python application where you intend to interact with the Temporal Cloud API.
+
+2. **Utilize the API:**
+   - Use the classes and methods defined in the imported files to communicate with the Temporal Cloud services.
+   - Ensure to handle any required authentication or configuration as needed for Temporal Cloud.
+
+This approach can be adapted for other programming languages by following their respective import and usage conventions for the generated code files.
+
+### Use the API
+
+When interacting with the Temporal Cloud Ops API, follow these guidelines:
+
+1. **API Version Header:**
+   - Always include the `temporal-cloud-api-version` header in your requests, specifying the API version identifier.
+   - The current API version can be found [here](https://github.com/temporalio/api-cloud/blob/main/VERSION#L1C1-L1C14).
+
+2. **Connection URL:**
+   - Connect to the Temporal Cloud using the gRPC URL: `saas-api.tmprl.cloud:443`.
+
+3. **Engagement Steps:**
+   - **Generate API Key:**
+     - Obtain an API Key for authentication. Note that many operations may require Admin privileges.
```

Comment:
add link to Generate API Keys docs https://temporal-documentation-git-api-key.preview.thundergun.io/cloud/api-keys#generate-an-api-key